site/content/en/docs/tasks/run/statefulset.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Run StatefulSet"
+linkTitle: "StatefulSet"
+date: 2024-07-25
+weight: 6
+description: >
+  Run a StatefulSet as a Kueue-managed workload.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management
+capabilities when running StatefulSets.
+Although Kueue does not yet support managing a StatefulSet as a single Workload,
+it's still possible to leverage Kueue's scheduling and resource management capabilities for the individual Pods of the StatefulSet.
+
+We demonstrate how to support scheduling StatefulSets in Kueue based on the Plain Pod integration,
+where every Pod from a StatefulSet is represented as a single independent Plain Pod.
+
+This guide is for [serving users](/docs/tasks#serving-user) that have a basic understanding of Kueue.
+For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).
+
+2. Follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
+   to learn how to enable the `v1/pod` integration and how to configure it using the `podOptions` field.
```

Comment:
I think it would be great to mention that we need to include "statefulset" in the "frameworks" section too.